### PR TITLE
Streaming results

### PR DESCRIPTION
**Pull request by bserdar**

### CODE REVIEW

File: http/src/main/java/com/redhat/lightblue/client/http/LightblueHttpClient.java [hunk 1]
```diff
@@ -268,6 +276,101 @@ public void close() throws IOException {
...
+                            }
+                        };
+                    streamParser.parse();
+                    
+                } catch (IOException e) {
+                    throw new LightblueException(e);
+                } finally {
+                    try {
+                        stream.close();
+                    } catch (Exception e) {}
```
Comment (bvulaj) [inline http/src/main/java/com/redhat/lightblue/client/http/LightblueHttpClient.java]: Worth logging something?

**Comment by bserdar** [comment]
Added logging

**Comment by alechenninger** [comment]
AFAIK you should be able to stream the parsing of the HTTP response regardless of whether lightblue is streaming results as it iterates over the cursor or as it serializes to JSON. To the client, it's chunked encoding JSON either way.

**Comment by bserdar** [comment]
Is that really sent as chunked-encoding? I think I got responses with fixed
content-lengths (not chunked-encoding).

**Pull request merged**